pyxem/signals/diffraction2d.py

```diff
@@ -404,7 +404,7 @@ def subtract_diffraction_background(
         if method == "h-dome":
             self.data = self.data / np.max(self.data)
             bg_subtracted = self.map(regional_filter, inplace=False, **kwargs)
-            bg_subtracted.map(filters.rank.mean, selem=square(3))
+            bg_subtracted.map(filters.rank.mean, footprint=square(3))
```

Comment:
`selem` was changed to `footprint` in scikit-image 0.18.0 as far as I can tell (https://scikit-image.org/docs/0.18.x/api/skimage.filters.rank.html#skimage.filters.rank.mean). Should bump minimal version of scikit-image to 0.18.0.